README.md

```diff
@@ -8,11 +8,82 @@ Airkeeper will fetch the value from the API (similarly to Airnode) and will also
 
 ## Setup
 
-- Airkeeper will require a configuration file that matches the one being used by the Airnode that should be used to update the beacon server value. You can just copy over the config.json file from the Airnode repo and put it in the /config directory of this repo.
+- Airkeeper will require a configuration file that matches the one being used by the Airnode that should be used to update the beacon server value. You can just copy over the config.json file from the Airnode repo and put it in the /config directory of this repo. Same goes for sectrets.env file from the Airnode repo. Examples of these two files can be found in the /config directory of this repo.
 
 - Airkeeper will also require an additional configuration file that will be merged with the one mentioned above and it will contain the configuration specific to Airkeeper. This file needs to be called airkeeper.json and you can find an example in the /config directory of this repo.
 
-## Instructions
+- Another requirement is to have an AWS account and cloud provider credentials must be provided in the aws.env file. An example of this file can be found in the /config directory of this repo.
+
+## Docker instructions
+
+Use the docker image to deploy or remove an Airkeeper to and from a cloud provider such as AWS.
+
+The docker image supports two commands.
+
+- `deploy`: Deploys an Airkeeper using configuration files.
+- `remove`: Removes an Airkeeper previously deployed.
+
+### Build docker image
+
+You can build the docker image by running the following command from the root directory:
+
+```
+docker build . -t api3/airkeeper
+```
+
+### deploy command
+
+The `deploy` command will create a new AWS lambda function and a new AWS cloud scheduler.
+
+```sh
+docker run -it --rm \
+--env-file config/aws.env \
+--env COMMAND=deploy \
+-v "$(pwd)/config/airkeeper.json:/airkeeper/config/airkeeper.json" \
+-v "$(pwd)/config/config.json:/airkeeper/config/config.json" \
+-v "$(pwd)/config/secrets.env:/airkeeper/config/secrets.env" \
```

Comment:
Since all the files are in the same directory that is mapped to the same directory in the container the only volume definition you need is mapping that directory: `"$(pwd)/config:/airkeeper/config"`.
But if you want to stress that these three files are needed it's fine.

Reply:
I initially started running the commands like that since I saw it [here](https://github.com/api3dao/api3-docs/blob/main/docs/airnode/v0.4/grp-providers/docker/deployer-image.md#L80) but then I realized that all the *.example files were also in that directory and I wasn't sure if I should continue to map the whole directory. To be honest I like your suggestion anyways since it's easier for the user to run the commands